exercises/package.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
I think we don't need this file. `package.json` in the root folder is ok. `package.json` on each exercise folder is ok. But this is not in the root folder not in exercise specific folder. I think it could be deleted.

Reply:
Good catch-- that's an artifact of a broken 'make' run.  I'll remove it.